app/components/file_attachment_list/file_attachment_document.js

```diff
@@ -24,19 +24,27 @@ import FileAttachmentIcon from './file_attachment_icon';
 
 const {DOCUMENTS_PATH} = DeviceTypes;
 export const SUPPORTED_DOCS_FORMAT = [
-    'application/pdf',
+    'application/json',
     'application/msword',
+    'application/pdf',
     'application/rtf',
     'application/vnd.ms-excel',
     'application/vnd.ms-powerpoint',
     'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
     'application/vnd.openxmlformats-officedocument.presentationml.presentation',
     'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+    'application/x-x509-ca-cert',
```

Comment:
This is an interesting document type to support -- is it likely that users will want to preview it vs. just download and double-click to install or some such? (Wondering if the preview might overwhelm less technical users who are asked to do something with the certificate file.)

Reply:
Good point, but as of now you cannot download the file in the mobile app